.github/workflows/tests.yaml

```diff
@@ -45,8 +45,9 @@ jobs:
         shell: micromamba-shell {0}        
         run: |
            python -V
+           coverage run --rcfile=coverage.toml -m pytest --verbose cstar/tests/*
            python -u -m coverage run --rcfile=coverage.toml tests/test_roms_marbl_example.py
```

Comment:
This PR now runs both the old and the new tests.

Reply:
Any chance we could separate these into two separate steps in the workflow? Hard to tell where one ends and the next begins

Reply:
Fair point. I actually should probably just make them separate CI jobs.